api/routes/v1/companies.js

```diff
@@ -0,0 +1,26 @@
+const router = require('express').Router()
+const Companies = require('../../models/companies')
+
+// Companies
+
+const companiesQuery = (key, value) => {
+  switch (key) {
+    case 'name':
+      return {name: new RegExp(value,'i')}
+    case 'employees_lte':
+      let lt = `employees.${value}`;
+      return {[`${lt}`]:{$exists: false}}
+    case 'employees_gte':
+      let gt = `employees.${value}`;
+      return {[`${gt}`]:{$exists: true}}
+  }
+}
+
+router.get('/', async (req, res, next) => {
+  const status = 200
+  const query = (req.query ? companiesQuery(Object.keys(req.query)[0], req.query[Object.keys(req.query)[0]]) : '')
```

Comment:
`req.query` should always return `true` as it'll return an object. It also looks like you're only grabbing the first key in `req.query`, which could change depending on the request. Objects are not ordered.